CHANGELOG.md

```diff
@@ -154,6 +154,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (snapshots) [#14597](https://github.com/cosmos/cosmos-sdk/pull/14597) Move `snapshots` to `store/snapshots` and rename bump proto package to v1.
```

Comment:
We had no changelog for the package moving (I am noticing that we mostly forgot to a changelog all store refactors). Precising the renaming as well.